blocks/bricks/parallel.py

```diff
@@ -200,13 +208,18 @@ class Distribute(Fork):
         The names of the targets.
     source_name : str
         The name of the source.
+    target_dims : list
+        A list of target dimensions, corresponding to `target_names`.
+    source_dim : int
+        The dimension of the source input.
+    prototype : :class:`~blocks.bricks.Feedforward`, optional
+        The transformation prototype. A copy will be created for every
+        input. By default a linear transformation is used.
```

Comment:
Oh, it's the other way around, sorry. Maybe we should rename `Linear` to `Affine` and create `Linear` which is `Linear(use_bias=False)` using current terminology.